TransparentWall/HarmonyPatches/Patches/ObstacleControllerCullingLayer.cs

```diff
@@ -29,7 +32,7 @@ private static void Postfix(ref ObstacleController __instance)
 
             if (!Configuration.EnableForHeadset && !Configuration.DisableForLivCamera) return;
 
-            Renderer mesh = __instance.gameObject?.GetComponentInChildren<Renderer>(false);
+            var mesh = __instance.gameObject?.GetComponentInChildren<Renderer>(false);
```

Comment:
please use explicit typing
```suggestion
            Renderer mesh = __instance.gameObject?.GetComponentInChildren<Renderer>(false);
```